api/http/handler/hostmanagement/fdo/profiles.go

```diff
@@ -20,21 +26,237 @@ import (
 // @failure 500 "Server error"
 // @failure 500 "Bad gateway"
 // @router /fdo/profiles [get]
-func (handler *Handler) fdoProfiles(w http.ResponseWriter, r *http.Request) *httperror.HandlerError {
-	settings, err := handler.DataStore.Settings().Settings()
+func (handler *Handler) fdoProfilesList(w http.ResponseWriter, r *http.Request) *httperror.HandlerError {
+	profiles, err := handler.DataStore.FDOProfile().FDOProfiles()
 	if err != nil {
-		return &httperror.HandlerError{http.StatusInternalServerError, "Unable to retrieve settings from the database", err}
+		return &httperror.HandlerError{StatusCode: http.StatusInternalServerError, Message: err.Error(), Err: err}
```

Comment:
I know there's a linting warning, but soon we're planing to upgrade the library, then we can replace this with 
```return httperror.NewError(http.StatusInternalServerError, "Unable to retrieve settings from the database", err)```

so it will be easy to refactor those (just find and replace)

Reply:
done